backend/src/models/user.model.ts

```diff
@@ -0,0 +1,19 @@
+import { Schema, model } from 'mongoose';
+
+interface UserModel {
+    auth_id: string,
```

Comment:
that looks good to me. auth_id is for storing uid of the firebase user I presume.